test/gurka/test_match.cc

```diff
@@ -334,7 +335,7 @@ TEST(MapMatchRoute, IgnoreRestrictions) {
                                        },
                                        {{"type", "restriction"}, {"restriction", "no_straight_on"}}}};
   const gurka::map map =
-      gurka::buildtiles(layout, ways, {}, relations, "test/data/mapmatch_restrictions",
+      gurka::buildtiles(layout, ways, {}, relations, VALHALLA_BUILD_DIR "test/data/mapmatch_restrictions",
                         {{"mjolnir.timezone", VALHALLA_BUILD_DIR "test/data/tz.sqlite"}});
```

Comment:
i felt it made sense to do it manually in these cases where we use the config option overrides because if we rely on `make_config` doing the right thing by prefixing directories with `VALHALLA_BUILD_DIR`, but then we add a config override for timezone and we ourselves dont prefix it, then we've defeated ourselves.

alternatively i could have written code to see if any override value begins with "test/data" and then prefixed that but it felt a little too much? i dont feel stronly about it

Reply:
not sure I understand this: on line 205 you're removing the prefix for tzdb, on line 337 you're adding it for the graph build (which should be covered already by make_config) and on line 339 you left it again for tzdb. probably an oversight?

IMO it'd be more consistent that noone ever has to specify VALHALLA_BUILD_DIR manually, but it's also a much rarer thing to do, so I'm ok with it too.